client/src/commons/Forms/PlacesTypesAndSubTypes/PlacesTypesAndSubTypes.tsx

```diff
@@ -141,42 +129,51 @@ const PlacesTypesAndSubTypes: React.FC<Props> = (props: Props): JSX.Element => {
+                                            inputValue={placeSubTypeInput}
                                             getOptionSelected={(option) => option.id === props.value}
-                                            onChange={(event, chosenPlaceSubType) => {
-                                                setSelectedPlaceSubType(chosenPlaceSubType ? chosenPlaceSubType : undefined)
-                                                props.onChange(chosenPlaceSubType ? chosenPlaceSubType.id : '')
-                                            }}
-                                            onInputChange={(event, chosenPlaceSubType) => {
-                                                setSelectedPlaceSubTypeInput(chosenPlaceSubType);
-                                                if (chosenPlaceSubType === '') {
-                                                    props.onChange('');
+                                            onChange={(event, chosenPlaceSubType) => 
+                                                chosenPlaceSubType && onPlaceSubTypeChange(chosenPlaceSubType as PlaceSubType)
+                                            }
+                                            onInputChange={(event, placeSubTypeInput) => {
+                                                setPlaceSubTypeInput(placeSubTypeInput);
+                                                if (placeSubTypeInput === '') {
+                                                    onPlaceSubTypeChange(placesSubTypesByTypes[placeType][0] as PlaceSubType);
                                                 }
                                             }}
+                                            onBlur={props.onBlur}
+                                            placeholder={placeSubTypeDisplayName}
                                             renderInput={(params) =>
                                                 <TextField
                                                     {...params}
                                                     test-id='placeType'
-                                                    value={props.value ? props.value : ''}
-                                                    label={placeTypeDisplayName}
-                                                    onBlur={props.onBlur}
-                                                    id={InteractionEventDialogFields.PLACE_TYPE}
-                                                    placeholder={placeTypeDisplayName}
+                                                    label={placeSubTypeDisplayName}
+                                                   
                                                 />
                                             }
                                         />
                                     )}
                                 />
                                 :
                                 <Autocomplete
```

Comment:
maybe pass the component to const so you could use it twice without copy pasting?

Reply:
since its the same inside the controller